ui/lib/apps/SlowQuery/pages/Detail/index.tsx

```diff
@@ -116,7 +124,14 @@ function DetailPage() {
                               expanded={detailExpand.prev_query}
                               onClick={togglePrevQuery}
                             />
-                            <CopyLink data={data.prev_stmt!} />
+                            <CopyLink
+                              textTransKey="copyFormatted"
```

Comment:
maybe better to make it type safe, i.e. `<CopyLink displayVariant={undefined|"default"|"original_sql"|"formatted_sql"}>`

Reply:
Make sense!